openedx/repo-tools
# fix: code refactored to add new ecosystem

I have created this PR to refactor dependabot_yml.py script that would check if a dependabot.yml config exist in repo then add new ecosystem github-action in it and don't override existing one, if not then create new file and add that ecosystem in it.

**How to test:**
Clone a repo let say eddx-repo-health and then install repo tools dependencies and run the following command to test if it is working fine, you should verify the changes after running this command:

`python PATH_TO_REPO_TOOLS_DIR/edx_repo_tools/dependabot_yml.py --path .github/dependabot.yml`
